easybuild/easyconfigs/n/nvtop/nvtop-1.0.0-fosscuda-2018b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'nvtop'
+version = '1.0.0'
+
+homepage = 'https://github.com/Syllo/nvtop'
+description = 'htop-like GPU usage monitor'
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['https://github.com/Syllo/nvtop/archive/1.0.0.tar.gz']
```

Comment:
this should be `sources = ['%(version)s.tar.gz']`

Reply:
Fixed and also removed the explicit CUDA dependency since that's already implied by 'fosscuda'.